src/components/settings/based.tsx

```
@@ -12,9 +14,43 @@
+  const [backgroundFiles, setBackgroundFiles] = useState<string[]>([])
+  const backgroundImageUrl = homeStore((s) => s.backgroundImageUrl)
+
+  useEffect(() => {
+    fetch('/api/get-background-list')
+      .then((res) => res.json())
+      .then((files) => setBackgroundFiles(files))
+      .catch((error) => {
+        console.error('Error fetching background list:', error)
+      })
+  }, [])
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)